src/controllers/moderationactions/ModerationActionModel.hpp

```diff
@@ -13,6 +13,11 @@ class ModerationActionModel : public SignalVectorModel<ModerationAction>
 public:
     explicit ModerationActionModel(QObject *parent);
 
+    enum Column {
```

Comment:
This should probably be an `enum class`.

Reply:
An `enum` non-`class` is implicitly convertible to int. This is really useful here as these are used for indexing a vector.